run-genny

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
```

Comment:
Just nosing in with a thought, feel free to ignore, but one of the original goals behind the `./scripts/genny` script in the long-running workloads project was to make the fact that it's a shell script running python running a binary as invisible as possible, since most people just want to run "genny".

Would it make sense to preserve that illusion and rename this to `genny`, keep passing top-level commands through to the binary, and have some sort of `lamp` subcommand that exposes the rest of the script? Since the bread-and-butter of invoking this would be to run the genny binary anyway?

Totally optional opinion though, feel free to ignore.

Reply:
Good idea. I'm going to revamp this a bit to be `./genny` that forwards args to `genny_core` except for subcommands that it knows about (or something like that).